.github/workflows/check.yml

```
@@ -34,6 +34,9 @@ jobs:
         with:
           persist-credentials: false
 
+      - uses: wimpysworld/nothing-but-nix@v6
+        if: ${{ runner.os == 'Linux' }}
```

I'd prefer to use the least aggressive option we can get away with, also helps avoid https://github.com/wimpysworld/nothing-but-nix/issues/18
```suggestion
      - uses: wimpysworld/nothing-but-nix@v6
        if: runner.os == 'Linux'
        with:
          hatchet-protocol: holster
```